script/ci

```diff
@@ -2,6 +2,10 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+run_linter() {
+  bundle exec rubocop
```

Comment:
I don't know much about travis (I use gitlab), but I think the linter should be a test on its own, as to not be run for every ruby/jruby/other version that the gem is tested against. The result won't be different on different interpreters.